.prettierrc

```diff
@@ -1,9 +1,18 @@
 {
-  "printWidth": 100,
+  "printWidth": 120,
```

Comment:
this isn't mentioned in PR description. is this accidental ?

Reply:
not accidental.. updated PR description